### PR TITLE
New: Common types package

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

File: packages/utils-types/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Set of common types used in webhint.
+
+## Installation
+
+To install the package, you need to run:
+
+```bash
+npm install @hint/utils-types
+```
```
Comment (molant) [inline packages/utils-types/README.md]: Use the same language than other packages:

> This package is installed automatically by webhint:
> 
> `npm install hint --save-dev`

I guess this applies to all the other packages you have in another PR.

File: packages/utils-types/src/severity.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/** The severity configuration of a hint */
+export enum Severity {
+    off = 0,
+    warning = 1,
+    error = 2
```
Comment (molant) [inline packages/utils-types/src/severity.ts]: Gonna have to update my Severity PR 😭

Comment (sarvaje) [reply packages/utils-types/src/severity.ts]: hehehe

File: packages/utils-types/README.md (hunk 1 above)
Comment (sarvaje) [reply packages/utils-types/README.md]: done!

Yep, I will need to update the other packages in the other PR :)

**Comment by sarvaje** [comment]
@molant I think we can close this PR, we don't need to publish this before the flattened PR.

**Pull request closed without merging**